### DISCUSSION TITLE
What's the release version with bazel supported?

### DISCUSSION THREAD

**Discussion by sunzhec2702** [eclipse-iceoryx/iceoryx#2213] [General]
Dear all,

The master branch already supports Bazel now. May I know what's the latest release version/tag to support Bazel?

**Comment by mossmaurice** [comment]
Hi @sunzhec2702 

Thanks for your question. Bazel is currently only available on `master` and was not released with `v2.x` Blueberry :blueberries: yet

However, I have good news for you. The `v3.0` C-release is right around the corner and is likely to land in March 2024. You can find more details on the timeline in this discussion: https://github.com/eclipse-iceoryx/iceoryx/discussions/2189

**Reply by sunzhec2702** [reply]
Thanks for your quick and nice reply. 

Let me use the 2.90.0 as a release candidate and wait for the final release of v3.0.
